docs/tutorials/tutorial-0.rst

```diff
@@ -64,7 +66,7 @@ It is all ready for using ``briefcase``. You can invoke it, using:
 
 .. code-block:: bash
 
-    $ python setup.py ios
+    $ python setup.py ios OR linux
```

Comment:
This is potentially confusing - do I type `python setup.py ios OR linux`, or do I type `python setup.py ios` or `python setup.py linux`? It would be better to break this out into 2 separate commands, describing which one you run depending on which platform you're on.